cli/src/command.rs

```diff
@@ -215,6 +215,68 @@ fn ensure_dev(spec: &Box<dyn service::ChainSpec>) -> std::result::Result<(), Str
 	}
 }
 
+/// Runs a performance check via compiling sample wasm code with a timeout.
+/// Only available in release build since the check would take too much time otherwise.
+/// Returns `Ok` if the check has been passed previously.
+#[cfg(not(debug_assertions))]
```

Comment:
CC

Reply:
Yet I think it makes sense to `cfg` the function body because it depends on kusama runtime being compiled in release mode, which essentially means `#[cfg(not(debug_assertions))]`
Otherwise `include_bytes` will raise compile error

Reply:
Looking at the code I think `fn host_perf_check` should always be called, and all the `cfg()` should be hidden inside.